### PR TITLE
Add public constructor for Acker

### PR DESCRIPTION
**Pull request by AbstractiveNord**
In some cases manual Acker creation is preferable, than usage `.split` method. That's public fn constructor introduction.

### CODE REVIEW

**Comment by Jarema** [comment]
Thanks for the PR!

Can you elaborate on use case for this?

File: async-nats/src/jetstream/message.rs [hunk 1]
```diff
@@ -336,6 +336,10 @@ pub struct Acker {
 // The async-trait crate is not a solution here, as it would mean we're allocating at every ack.
 // Creating separate function to ack just to avoid one duplication is not worth it either.
 impl Acker {
+    /// Public constructor
```
Comment (Jarema) [inline async-nats/src/jetstream/message.rs:339]: Please remove this comment.
It's obviously a public constructor :).

**Review by Jarema — CHANGES_REQUESTED** [review]
I'm ok merging this one, just please remove the comment.

File: async-nats/src/jetstream/message.rs (hunk 1 above)
Comment (AbstractiveNord) [reply async-nats/src/jetstream/message.rs]: Done.

**Review by Jarema — APPROVED** [review]
LGTM!

Thank you for your contribution!

**Pull request merged**